keyboards/e6_rgb/config.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * config.h
+ *
+ */
+
+#ifndef CONFIG_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```